recipe/meta.yaml

```diff
@@ -19,7 +19,7 @@ requirements:
   build:
     # dependencies needed for open source microsoft/vscode - https://github.com/microsoft/vscode/wiki/How-to-Contribute 
     # updating nodejs to >=20.0 as it's requirement for vscode - https://github.com/microsoft/vscode/wiki/How-to-Contribute
-    - nodejs >=20.0
+    - nodejs >=20.0,<21
```

Comment:
any specific reason we added `<21` ?

Reply:
vscode has dependency on @azure/msal-node@2.5 which expects node version 16,18, or 20.